### PR TITLE
LOGO now at center of the sidebar

### LINKED ISSUE #16418: Logo slightly to the left in sidebar.
### Description:

Rocket Chat logo is slightly to the left which looks a bit odd in the sidebar bottom.
### Steps to reproduce:

1. Open the app.
2. At the bottom of the sidebar, the logo is not centered.

### Expected behavior:
The logo must be centered.

Expected
### Actual behavior:
The logo is not centered, looks odd.
Actual

## Status: Working on it.

### PR DESCRIPTION
**Pull request by aknrg77**
Closes #16418 

I have fixed the logo , here's the SS 
Screenshot 2020-03-05 at 2 05 07 AM

### CODE REVIEW

File: app/theme/client/imports/components/sidebar/sidebar.css [hunk 1]
```diff
@@ -56,6 +56,7 @@
 		& img {
 			max-width: 100%;
 			height: 100%;
+			margin-left : 17%;
```
Comment (sampaiodiego) [inline app/theme/client/imports/components/sidebar/sidebar.css]: Codacy Issue found: [Unexpected whitespace before ":" (declaration-colon-space-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5130324)

Comment (aknrg77) [reply app/theme/client/imports/components/sidebar/sidebar.css]: ok i will fix it

**Comment by sampaiodiego** [comment]
I guess the logo position is by design, right @engelgabriel @brunosquadros ?

if so, the issue should be closed as well

**Comment by engelgabriel** [comment]
Hi @aknrg77 ,

Thank you for your contribution, it is much appreciated. 

But I am sorry I won't be able to merge this, as it is a matter of taste and our designer has aligned the logo to the left on purpose. 

Users can change the logo aliment using the custom CSS setting on the admin if they prefer. 

:)

**Pull request closed without merging**